src/SourceBuild/tarball/content/build.proj

```diff
@@ -119,7 +66,7 @@
 
   <Target Name="ReportPoisonUsage"
```

Comment:
Same thing, logged https://github.com/dotnet/source-build/issues/2317 for poisoning issues.